content/guides/getting-started/component-framework-configuration.md

```diff
@@ -2,22 +2,38 @@
 title: Framework Configuration
 ---
 
-Writing tests in Cypress is the same regardless of framework choice (Vue, React,
-Nuxt, Next, etc.). However, each framework requires additional setup and
-configuration to run properly.
+Writing tests in Cypress is the same regardless of whichever UI library you
+choose (React or Vue, currently). However, each framework requires a different
+setup configuration.
+
+Cypress currently supports the following frameworks for component testing:
+
+| Framework                                 | UI Library |
+| ----------------------------------------- | ---------- |
+| [Create React App](#Create-React-App-CRA) | React      |
+| [Next.js](#Next-js) (Alpha)               | React      |
```

Comment:
Was going to suggest this as well. You can just drop the <Badge /> component in here like it is in the [Migration Guide](https://cypress-documentation-git-udx-147-update-ct-c-5b2cf5-cypress-io.vercel.app/guides/references/migration-guide#Config-option-changes), etc. 

However, I'm now noticing there's something funky with the padding on these 😬  I added some a long time ago to the base badge element and got more specific with those E2E & CT content badges. I think these have just gotten used in a lot more places since then and that's not helping. Should be a small tweak if that's really the case.